Package.swift

```diff
@@ -5,7 +5,10 @@ let package = Package(
   name: "Lottie",
   // Minimum platform versions should be kept in sync with the per-platform targets in Lottie.xcodeproj, lottie-ios.podspec, and lottie-spm's Package.swift
   platforms: [.iOS("13.0"), .macOS("10.15"), .tvOS("13.0"), .custom("visionOS", versionString: "1.0")],
-  products: [.library(name: "Lottie", targets: ["Lottie"])],
+  products: [
+    .library(name: "Lottie", targets: ["Lottie"]),
+    .library(name: "Lottie-Dynamic", type: .dynamic, targets: ["Lottie"]),
```

Comment:
Can you add a comment here explaining the difference between Lottie and Lottie-Dynamic, and a quick explanation of when you would prefer to use Lottie-Dynamic instead of Lottie?